lagom-client-restaurant/menu-item-impl/src/main/resources/application.conf

```diff
@@ -1,10 +1,5 @@
 play.application.loader = com.example.menuitem.impl.MenuItemLoader
 
-http {
-  address = ${?HTTP_BIND_ADDRESS}
-  port = 9100
-}
-
```

Comment:
This probably never worked as expected, since when using Lagom's `runAll` (or `run`) it manages the port dynamically. This setting was overridden in `application.prod.conf` (but didn't need to be there, either).